dockers/docker-database/base_image_files/sonic-db-cli

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+DOCKER_EXEC_FLAGS="i"
+
+# Determine whether stdout is on a terminal
+if [ -t 1 ] ; then
+    DOCKER_EXEC_FLAGS+="t"
+fi
+
+dbid=`python -c "import swsssdk ; print swsssdk.SonicDBConfig.get_dbid(\"$1\")"`
+if [ $? -gt 0 ]; then
+    echo "Cannot get DB id for $1"
```

Comment:
>echo [](start = 4, length = 4)

Catch the exception and hide the stack output?

Reply:
rewrite with python